index.js

```diff
@@ -0,0 +1,66 @@
+const dropDown = document.querySelector('.dropdown-menu');
+const nav = document.querySelector('.disktop-nav');
+const closeBtn = document.querySelector('.close-btn');
+const speakersSection = document.querySelector('.speakers');
+
+dropDown.addEventListener('click', () => {
+  nav.style.display = 'unset';
+});
+
+closeBtn.addEventListener('click', () => {
+  nav.style.display = 'none';
+});
+
+const speakers = [
+  {
+    imgSrc: './imges/yahya-img.jpg',
+    name: 'yahya elganayni',
+    info: 'create the sites layout and integrate graphics, applications ',
+  },
+  {
```

Comment:
- [ ] Please don't forget about the missing elements I mentioned in the earlier comment above, and as a reminder here's a small hint 😉

```
{
  name: "Yahya Elganyni", 
  role: "Software Engineer", 
  description: "Some text are written here ...", 
  photo: "image_url_here"
}
```